README.md

```diff
@@ -1,56 +1,90 @@
 # Git Repository Basics
 
+
 ## Objectives
 
 1. Define Git and explain how it helps programmers control and manage changes
-2. Initialize a new Git repository with `git init`
-3. Check the state of your files using `git status`
-4. Stage files to commit using `git add` and `git commit`
-
-So, how do you use Git?
+2. State initializing Git repository with `git init`
+3. Use `git status` to check the status of your repository 
+4. Recognize how to keep track of files with `git add`
+5. Use `git commit` to apply a message to the commit
```

Comment:
Is it just "applying a message?" I think it's "to create a commit _and_ apply a commit message."

Keep in mind, they've not read the excellent context provided below yet. 

Objectives "prime" the readers as to what to look for so it needs to be written "down" in knowledge-level for where they're coming in. As a process step, writing them without reading the content means that you'll _probably_ default into the appropriate level for the reader.